packages/connect/src/relay/index.ts

```diff
@@ -106,18 +111,48 @@ class Relay {
     await this.libp2p.handle(CAN_RELAY_PROTCOL(this.options.environment), this._onCanRelay)
 
     this.webRTCUpgrader.start()
+
+    const periodicKeepAlive = async function (this: Relay) {
+      try {
+        await this.keepAliveRelayConnection()
+      } catch (err) {
+        log('Fatal error during periodic keep-alive of relay connections', err)
+      }
+    }.bind(this)
+
+    this.stopKeepAlive = retimer(
+      periodicKeepAlive,
+      // TODO: Make these values configurable
+      () => randomInteger(10000, 10000 + 3000)
+    )
   }
 
   setUsedRelays(peers: PeerId[]) {
     log(`set used relays`, peers)
     this.usedRelays = peers
   }
 
+  protected async keepAliveRelayConnection(): Promise<void> {
```

Comment:
Please document why this is needed.

Also the name is misleading since the actual ping isn't done.

Reply:
This is for the preparation that in future we'll add the actual ping, see the comment below.

Reply:
cf. the TODO comment in the function